### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by nonsequitur**

### CODE REVIEW

File: modules/home-environment.nix [hunk 1]
```diff
@@ -309,11 +309,11 @@ in
             relativePath="''${sourcePath#$oldGenFiles/}"
             targetPath="$HOME/$relativePath"
             if [[ -e "$newGenFiles/$relativePath" ]] ; then
-              $VERBOSE_ECHO "Checking $targetPath  exists"
+              $VERBOSE_ECHO "Checking $targetPath exists"
```
Comment (rycee) [inline modules/home-environment.nix]: My intention here was to have some form of separation between the "checking" part and the "result" part. But I agree that two spaces is silly. Would something like `"Checking $targetPath ... exists"` and `"Checking $targetPath ... gone (deleting)"` be more clear?

Comment (nonsequitur) [inline modules/home-environment.nix]: Yes, that would be fine, too.

Comment (nonsequitur) [inline modules/home-environment.nix]: After thinking this through, I'd prefer a colon: `Checking $targetPath: exists` :relaxed:

Comment (rycee) [inline modules/home-environment.nix]: Sure, that works as well.

**Comment by rycee** [comment]
Looking good. If you change the echo strings and reword the commits so that the two first start with "home-environment: " and the last with "lib-bash: " I'll be happy merge! 👍

**Comment by rycee** [comment]
Rebased into master ending in 5c098dc7ad2e8584b8799fd14dc79a40b86bae07. Many thanks for the cleanups!

**Pull request closed without merging**